server/routers/files/files_router.py

```
@@ -214,46 +216,103 @@ async def create_upload_files(
+    # well past any proxy timeout. Hand them to a background task and return
+    # now so the admin gets routed to a live progress page instead of a 504.
+    processing = bool(face_targets or keys)
+    if processing:
+        upload_jobs.start_job(
+            album_slug, face_detection=bool(face_targets), image_count=image_count
+        )
+        asyncio.create_task(
+            _process_album(
+                album.id, album_slug, face_targets, keys, update, image_count
+            )
+        )
```

Ruff (0.15.15)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)